src/components/settings-menu/BackupSection/AlreadyBackedUpView.js

```diff
@@ -145,6 +150,50 @@ export default function AlreadyBackedUpView() {
     });
   }, [walletId]);
 
+  const manageCloudBackups = useCallback(() => {
+    const buttons = ['Delete all iCloud backups', 'Cancel'];
+
+    showActionSheetWithOptions(
+      {
+        cancelButtonIndex: 1,
+        destructiveButtonIndex: 0,
+        options: buttons,
+        title: `Manage iCloud backups`,
+      },
+      buttonIndex => {
+        if (buttonIndex === 0) {
+          // Delete wallet with confirmation
+          showActionSheetWithOptions(
+            {
+              cancelButtonIndex: 1,
+              destructiveButtonIndex: 0,
+              message: `Are you sure you want to delete all iCloud Backups`,
+              options: ['YES, Delete All iCloud Backups', 'Cancel'],
+            },
+            async buttonIndex => {
+              if (buttonIndex === 0) {
+                const newWallets = { ...wallets };
+                Object.keys(newWallets).forEach(key => {
+                  delete newWallets[key].backedUp;
```

Comment:
if you don't really need to, it's better to set `undefined`.
```suggestion
                 newWallets[key] = undefined;
```

Reply:
https://levelup.gitconnected.com/5-facts-about-delete-operator-in-javascript-c16fd2588cd

it's generally a ton faster

Reply:
+ few other places